src/c++/perf_analyzer/test_concurrency_manager.cc

```diff
@@ -183,6 +183,20 @@ class TestConcurrencyManager : public TestLoadManagerBase,
     watchdog.stop();
   }
 
+  /// Test that idle time is tracked correctly
+  void TestOverhead()
+  {
+    stats_->SetDelays({1});
+    ChangeConcurrencyLevel(params_.max_concurrency);
+    std::this_thread::sleep_for(std::chrono::milliseconds(100));
+    // During a run of 100 ms (100,000,000 ns), make sure that the idle time is
+    // at least 95% of that
```

Comment:
Where did 95% come from?

Reply:
just a high bar for this simple test case where no prep work is needed.